### PR TITLE
[PT-3269] 500 errors to graylog

### PR DESCRIPTION
**Pull request by kirillbilchenko**

### CODE REVIEW

File: browsermob-dist/src/main/resources/bmp-logging.yaml [hunk 1]
```diff
@@ -8,30 +8,52 @@ configuration:
...
-      -
-        name: file
-        fileName: bmp.log
-        PatternLayout:
-          pattern: "[%-5level %date{ISO8601} %logger] (%thread) %msg %xThrowable%n"
-        append: false
+        pattern: "[%-5level %date{ISO8601} %c{1}] %msg %xThrowable%n"
+    gelf:
+      - name: gelf
+        host: udp:graylog000.staging.hellofresh.io
```
Comment (CVirus) [inline browsermob-dist/src/main/resources/bmp-logging.yaml]: Please use tcp on graylog-input.staging.hellofresh.io

File: browsermob-dist/src/main/resources/bmp-logging.yaml [hunk 2]
```diff
@@ -8,30 +8,52 @@ configuration:
...
+        port: 12201
```
Comment (CVirus) [inline browsermob-dist/src/main/resources/bmp-logging.yaml]: Please use port 12204

**Review by antonu17 — DISMISSED** [review]
I have a big concern of publishing any hellofresh-related configuration to this repository. I'm talking about `browsermob-dist/src/main/resources/bmp-logging.yaml` file here. As this is a public fork, let's keep just an example configuration here. The real config file rendering must be implemented in corresponding automation project.

**Comment by antonu17** [comment]
Great job of making browsermob-proxy publishing logs to Graylog! However, let's keep this project free from hellofresh-related assets.

**Pull request merged**